packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1886,58 +2319,146 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       expect(service.toObject(regularSelection['valueId']! as String), richTextDiagnostic.value);
     }, skip: !WidgetInspectorService.instance.isWidgetCreationTracked()); // [intended] Test requires --track-widget-creation flag.
 
-    testWidgets('ext.flutter.inspector creationLocation', (WidgetTester tester) async {
-      await tester.pumpWidget(
-        Directionality(
-          textDirection: TextDirection.ltr,
-          child: Stack(
-            children: const <Widget>[
-              Text('a'),
-              Text('b', textDirection: TextDirection.ltr),
-              Text('c', textDirection: TextDirection.ltr),
-            ],
-          ),
-        ),
-      );
-      final Element elementA = find.text('a').evaluate().first;
-      final Element elementB = find.text('b').evaluate().first;
+    testWidgets('ext.flutter.inspector creationLocation', (WidgetTester tester) async {
```

Comment:
NOTE: this diff looks a little weird here because i changed `service.setPubRootDirectories(<String>[])` to `service.resetPubRootDirectories(<String>[])`.

Nothing else changed inside the `creationLocation` test though